api/src/database/migrations/run.ts

```diff
@@ -14,7 +14,7 @@ export default async function run(database: Knex, direction: 'up' | 'down' | 'la
 	let customMigrationFiles =
 		((await fse.pathExists(customMigrationsPath)) && (await fse.readdir(customMigrationsPath))) || [];
 
-	migrationFiles = migrationFiles.filter((file: string) => file.startsWith('run') === false && file.endsWith('.js'));
+	migrationFiles = migrationFiles.filter((file: string) => /^[0-9]+[A-Z]-[^.]+\.(?:js|ts)$/.test(file));
```

Comment:
How does this handle non-core custom migration IDs? For example "rijk-1.my_migration.js"

Reply:
Aren't they handled one line below?